didgen/inverter.py

```diff
@@ -221,7 +221,7 @@ def keep_in(data):
         fea_h, adj_vec = start_from(dataset[qmid].to(device), config)
 
         mask = (adj_vec == 0)
-        mask_mask = (torch.rand(mask.shape) > 0.15)
+        mask_mask = (torch.rand(mask.shape) > 0.15).to(device)
         config.adj_mask = ~(mask * mask_mask)
```

Comment:
Could you remove these three lines? They are from an old version and are never used. I was not able to push to your branch to make the changes myself.